apps/dashboard/app/assets/stylesheets/application.scss

```diff
@@ -101,15 +101,17 @@ pre.motd-monospaced {
 
 // Add branded panels
 
-/**
-.panel-ood-blue {
-  @include panel-variant($ood-blue, $ood-white, $ood-blue, $ood-blue);
+.sandboxAppsHeader {
+  background-color: $ood-burgundy;
+  color: $ood-white;
 }
 
-.panel-ood-burgundy {
-  @include panel-variant($ood-burgundy, $ood-white, $ood-burgundy, $ood-burgundy);
+.systemAppsHeader {
+  background-color: $ood-blue;
+  color: $ood-white;
 }
```

Comment:
Does just a `s/panel/card` work here? You'll have to excuse my ignorance of css variants here.

Whatever we do, I think the names should continue to be snake case with hyphens instead of camel case.

Reply:
Agreed on the naming, I'll make that change. 

The bootstrap docs specifically called out panels as being no longer supported which is why I removed them altogether since we are using bs 4.6 according to the package.json. Here is the relevant bootstrap doc: https://getbootstrap.com/docs/4.6/components/card